### PR TITLE
frontend: Hook up experimental operators in Terraform GUI.

### PR DESCRIPTION
**Pull request by ggreer**
Also, fix the app id for all platforms.

### CODE REVIEW

File: installer/frontend/cluster-config.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ export const DEFAULT_CLUSTER_CONFIG = {
   [UPDATER]: {
     server: 'https://tectonic.update.core-os.net',
     channel: 'tectonic-1.5',
-    appID: '',
+    appID: '6bc7b986-4654-4a0f-94b3-84ce6feb1db4',
```
Comment (amrutac) [inline installer/frontend/cluster-config.js]: Where does this come from?

File: installer/frontend/cluster-config.js [hunk 2]
```diff
@@ -386,6 +386,7 @@ export const toAWS_TF = (cc, FORMS) => {
       tectonic_worker_count: workers[NUMBER_OF_INSTANCES],
       // TODO: shouldn't hostedZoneID be specified somewhere?
       tectonic_dns_name: cc[CLUSTER_SUBDOMAIN],
+      tectonic_experimental: cc[UPDATER_ENABLED],
```
Comment (amrutac) [inline installer/frontend/cluster-config.js]: Should there be a separate check if the customer is using the old installer?

File: installer/frontend/cluster-config.js (hunk 1 above)
Comment (yifan-gu) [inline installer/frontend/cluster-config.js]: @amrutac This is an hardcoded value in the coreupdate server.

File: installer/frontend/cluster-config.js (hunk 2 above)
Comment (ggreer) [inline installer/frontend/cluster-config.js]: Hmm... I'm not sure what you mean. This function (`toAWS_TF()`) is only called if using the terraform installer. The old cloudformation behavior remains the same.

Comment (amrutac) [inline installer/frontend/cluster-config.js]: Oh OK, just wanted to make sure this doesn't apply for the old cloudformation clusters.

File: installer/frontend/cluster-config.js (hunk 1 above)
Comment (amrutac) [inline installer/frontend/cluster-config.js]: OK.

**Pull request merged**